languages.toml

```
@@ -2698,3 +2699,27 @@ file-types = ["webc"]
 roots = []
 indent = { tab-width = 2, unit = "  " }
 grammar = "html"
+
+[[language]]
+name = "unison"
+scope = "source.unison"
+injection-regex = "unison"
+file-types = ["u"]
+shebangs = []
+roots = []
+auto-format = false
+comment-token = "--"
+indent = { tab-width = 4, unit = "    " }
+language-servers = ["ucm"]
+
+[language.auto-pairs]
+'(' = ')'
+'{' = '}'
+'[' = ']'
+'"' = '"'
+'`' = '`'
+
+[[grammar]]
+name = "unison"
+source = { git = "https://github.com/kylegoetz/tree-sitter-unison", branch = "deploy", rev = "1df9e35477b4261abb7cd9d3ab5b2f565fc3fd8a" }
```

```suggestion
source = { git = "https://github.com/kylegoetz/tree-sitter-unison", rev = "1df9e35477b4261abb7cd9d3ab5b2f565fc3fd8a" }
```

only `git`, `rev` and `subpath` are supported https://github.com/helix-editor/helix/blob/d6856cfeec14305b3405a6aee90c73c80b41e48f/helix-loader/src/grammar.rs#L51-L57